src/components/paragraphs/sup-carousel/sup-carousel-paragraph.tsx

```diff
@@ -65,71 +65,75 @@ const Slide = ({slideParagraph, isTopHero}: { slideParagraph: ParagraphSupCarous
   return (
     <article
       aria-labelledby={slideParagraph.id}
-      className={twMerge("relative centered-container text-white", clsx({
+      className={twMerge("relative centered-container text-white w-full h-full min-h-full lg:min-h-screen", clsx({
```

Comment:
`lg:min-h-screenl`? this makes it fill the entire screen size. Is that actually desired? It creates a large unused space below the text contents